docs/ops.html

```diff
@@ -1373,6 +1373,27 @@ <h5 class="anchor-heading"><a id="ext4" class="anchor-link"></a><a href="#ext4">
     <li>delalloc: Delayed allocation means that the filesystem avoid allocating any blocks until the physical write occurs. This allows ext4 to allocate a large extent instead of smaller pages and helps ensure the data is written sequentially. This feature is great for throughput. It does seem to involve some locking in the filesystem which adds a bit of latency variance.
   </ul>
 
+  <h4 class="anchor-heading"><a id="replace_disk" class="anchor-link"></a><a href="#replaced_disk">Replace KRaft Controller Disk</a></h4>
+  <p>When Kafka is configured to use KRaft instead of ZooKeeper, the controllers stores the cluster metadata in the directory specified in <code>metadata.log.dir</code>, <code>log.dir</code> or <code>log.dirs</code>. See the documentation for <code>metadata.log.dir</code> for details.</p>
```

Comment:
I'd suggest "the controllers stores the cluster metadata in the directory specified in metadata.log.dir -- or the first log directory, if metadata.log.dir is not configured."

Reply:
Done. Thanks for the suggestion.